Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "msquic"
-version = "2.1.0-beta"
+version = "2.0.1-beta"
```

Comment:
Undo this please.

Reply:
I even see it when preparing the new branch but then I went too fast...